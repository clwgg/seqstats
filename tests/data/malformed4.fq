@seq1
ACGT
+
!!!!
@seq2
TGCA