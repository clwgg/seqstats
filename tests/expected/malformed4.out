FASTQ quality string length (0) does not match sequence length (4).
